debezium-core/src/main/java/io/debezium/bean/StandardBeanNames.java

```diff
@@ -42,4 +43,9 @@ public interface StandardBeanNames {
     // all connector-specific connection details across all connectors (including MongoDB),
     // and then replace this name key as CONNECTION_FACTORY regardless of the connector.
     String JDBC_CONNECTION = JdbcConnection.class.getName();
```

Comment:
If this is supposed to include Mongo, can we use something like `DatabaseConnection` as a name? IMHO even when documented this is going to be very unintuitive .

Reply:
Actually this is only for relation databases as of now.

Reply:
I think that's reasonable @jcechace, although I'd like to find time to explore the comment above that to see if there is some type of unification we can do across MongoDB and Relational connectors.